### ISSUE TITLE
Zoom to multiple elements

### ISSUE THREAD

**Issue by fromi** [BetterTyped/react-zoom-pan-pinch#388]
**Is your feature request related to a problem? Please describe.**
I need to zoom to multiple elements for the tutorials on [Game Park](https://game-park.com/).
For example, I want to zoom to the cards inside the hand of the players. Cards are not inside a parent component, so I need to zoom to all of them.

I was able to implement the feature here: https://github.com/gamepark/react-game/blob/main/src/hooks/useZoomToElements.ts
It is ugly because I had to copy a lot of utility functions from react-zoom-pan-pinch.

**Describe the solution you'd like**
Either a new function "useZoomToElements, or extends the existing "useZoomToElement" function to allows an array of nodes as the parameter.

**Describe alternatives you've considered**
I could have created an invisible box on the fly covering all the desired elements, but it is a lot harder to implement and feel hacky.

I can make a pull request if you describe which API solution you prefer.

**Comment by maths1009** [comment]
I agree with your suggestion to be able to pass a node list or an id list. 

I don't know if elements placed in abosulte pose a problem for this functionality.

**Comment by fromi** [comment]
I implemented the solution on my framework by copying large amount of code from this repo.
You can test it here, playing the tutorial of Expedition board game: https://expedition.game-park.com/ 
Works like a charm!

**Comment by fromi** [comment]
> How did you do it? It works great!

This way: https://github.com/gamepark/react-game/blob/main/src/hooks/useZoomToElements.ts
Mostly copy pasting of that repository as I said.

**Comment by maths1009** [comment]
up

**Comment by prc5** [comment]
Shipped: `zoomToElement` accepts an element, an element id, or an array of either, and frames the union of their boxes: `zoomToElement([cardA, cardB], { animationTime: 300 })`. Ids that do not resolve are skipped.

---
_Shipped on master (commits 3ab2a4a / ef03a03 / fad10f1 / 9e5e10d, 2026-09-01). It goes out with the next semantic-release, which CI publishes from master; watch the Releases page for the version number._

**Comment by fromi** [comment]
Awesome @prc5 , thanks! We still use the feature heavily on Game Park, now in 50+ games :)
